nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -374,6 +381,222 @@ in {
         '';
       };
 
+      defaultLocale = mkOption {
+        default = null;
+        type = types.nullOr types.str;
+        example = "de_CH";
+        description = ''
+          <warning>
+           <para>This option exists since Nextcloud 21! If older versions are used,
+            this will throw an eval-error!</para>
+          </warning>
+
+          Default locale to use when Nextcloud fails to detect the user's
+          locale.
+
+          Further details about this setting and supported locale codes can be found in the
+          <link xlink:href="https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/language_configuration.html">upstream documentation</link>.
+        '';
+      };
+
+      twofactorEnforced = let
+        twoFactorAuthUpstreamManualUrl = "https://docs.nextcloud.com/server/latest/admin_manual/configuration_user/two_factor-auth.html";
+      in {
+        enabled = mkOption {
+          default = null;
+          type = types.nullOr types.bool;
+          example = true;
+          description = ''
+            ${uiClashWarning}
+
+            Whether to enforce two-factor authentication.
+
+            Further details about this setting and supported locale codes can be found in the
+            <link xlink:href="${twoFactorAuthUpstreamManualUrl}">upstream documentation</link>.
```

Comment:
Perhaps define this entire paragraph as a variable rather than just the URL, since it is all repeated.